packages/core/src/Stores/traders-hub-store.js

```diff
@@ -672,86 +671,14 @@ export default class TradersHubStore extends BaseStore {
         });
     }
 
+    updateExchangeRates = data => {
+        this.exchange_rates_2 = data;
+    };
+
     toggleAccountTransferModal() {
         this.is_account_transfer_modal_open = !this.is_account_transfer_modal_open;
     }
 
-    async updatePlatformBalance() {
-        runInAction(() => {
-            this.is_balance_calculating = true;
-        });
-        const { accounts, dxtrade_accounts_list, mt5_login_list } = this.root_store.client;
-
-        const account_list = Object.keys(accounts).map(loginid => accounts[loginid]);
-        const platform_demo_account = account_list.find(account => account.is_virtual);
-        if (platform_demo_account) {
-            const { balance, currency } = platform_demo_account;
-            this.platform_demo_balance = { balance, currency };
```

Comment:
@shayan-deriv Make sure to delete and replace everything that was deleted here, For example, the `is_balance_calculating` observer needs to be deleted from this store, or `platform_demo_balance` needs to be replaced in `DemoAccountCard` with the hook, And we need to remove them from the type file too.

Reply:
sure man. I've already refactored `DemoAccountCard` with the new hooks. will check the rest as well